torch/_torch_docs.py

```diff
@@ -2719,33 +2719,51 @@ def merge_dicts(*dicts):
     other (Tensor or Number): the divisor
 
 Keyword args:
+    rounding_mode (str, optional): Type of rounding applied to the result:
+
+        * ``"true"`` - default behavior. Performs no rounding and, if both :attr:`input` and
+          :attr:`other` are integer types, promotes the inputs to the default scalar type.
+          Equivalent to true division in Python (the ``/`` operator) and NumPy's ``np.true_divide``.
+        * ``"trunc"`` - rounds the results of the division down.
```

Comment:
The description for trunc and floor are identical, "rounds the results of the division down". 

For trunc I think we can say rounds towards zero?

https://github.com/pytorch/pytorch/pull/50280#discussion_r555070519